examples/README.md

```diff
@@ -117,6 +117,7 @@ relevant code.
 - [SIR Epidemiology model](./toy/model-sir.ipynb)
 - [Stochastic Degradation model](./toy/model-stochastic-degradation.ipynb)
 - [Stochastic Logistic model](./toy/model-stochastic-logistic-growth.ipynb)
+- [Michaelis Menten model](./toy/model-stochastic-michaelis-menten.ipynb)
```

Comment:
Separate category here too?

Reply:
Yes, I think separate out the stochastic ones into their own subsection of "toy models".